app/src/main/res/drawable/ic_right_facing_arrow.xml

```diff
@@ -0,0 +1,36 @@
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item>
+        <shape>
+            <solid android:color="@android:color/transparent"/>
+            <size android:width="2dp" android:height="50dp"/>
+        </shape>
+    </item>
+
+    <item android:bottom="20dp">
```

Comment:
dimen resource

Reply:
Its a drawable. Do we need to us dimen resource here too?

Reply:
No need

Reply:
@iamareebjamal 👍 
Please merge if this is fine now.